README.md

```diff
@@ -40,8 +40,6 @@ Please open a github issue including cast version number `cast --version`.
 
 Pull requests are gratefully received!
 
-- please 'gofmt' the code.
```

Comment:
Not sure why you're removing this request for PRs to be gofmt'ed?

Reply:
Ohhhh, sorry. I thought the README was requesting someone gofmt the existing code. I’ll put that back.